From 0c4d8a5ad64a69e8c8205035f7b002ff263b902a Mon Sep 17 00:00:00 2001
From: Yu Yue <yuyue@mprc.pku.edu.cn>
Date: Mon, 31 Oct 2011 17:58:45 +0800
Subject: [PATCH 052/641] UniCore64: Add function init_IRQ

Fixed error:
noinitramfs.c:(.init.text+0xb38): undefined reference to `init_IRQ'

Signed-off-by: Yu Yue <yuyue@mprc.pku.edu.cn>
---
 arch/unicore64/kernel/Makefile |    2 +-
 arch/unicore64/kernel/irq.c    |   11 +++++++++++
 2 files changed, 12 insertions(+), 1 deletion(-)
 create mode 100644 arch/unicore64/kernel/irq.c

diff --git a/arch/unicore64/kernel/Makefile b/arch/unicore64/kernel/Makefile
index ea2c5c0..eea7b84 100644
--- a/arch/unicore64/kernel/Makefile
+++ b/arch/unicore64/kernel/Makefile
@@ -1,4 +1,4 @@
 obj-y			:= ptrace.o setup.o init_task.o process.o
-obj-y			+= time.o traps.o
+obj-y			+= time.o traps.o irq.o
 head-y			:= head.o
 extra-y			:= $(head-y) vmlinux.lds
diff --git a/arch/unicore64/kernel/irq.c b/arch/unicore64/kernel/irq.c
new file mode 100644
index 0000000..33908e6
--- /dev/null
+++ b/arch/unicore64/kernel/irq.c
@@ -0,0 +1,11 @@
+#include <linux/kernel.h>
+#include <linux/init.h>
+
+/**
+ * init_IRQ() -
+ */
+void __init init_IRQ(void)
+{
+	/* FIXME */
+	BUG();
+}
-- 
1.7.9.5

